src/app/components/team/Newsletter/NewsletterRssFeed.js

```diff
@@ -77,7 +77,13 @@ const NewsletterRssFeed = ({
                     placeholder={placeholder}
                     disabled={disabled}
                     className={styles['rss-feed-url-field']}
-                    onChange={(e) => { setLocalRssFeedUrl(e.target.value); }}
+                    onChange={(e) => {
+                      let { value } = e.target;
+                      if (!/^https?:\/\//.test(value)) {
+                        value = `https://${value}`;
```

Comment:
I can't confirm from my phone but will this prepend https to an invalid url too? If yes, maybe check clientside whether appending https:// creates a valid URL, and if not, then don't actually commit the prepend

Reply:
It's almost like a prefix... the backend will return an error if it's not valid.